torch/csrc/distributed/rpc/python_functions.cpp

```diff
@@ -35,36 +31,30 @@ std::shared_ptr<FutureMessage> py_rpc_builtin(
     const std::string& opName,
     const py::args& args,
     const py::kwargs& kwargs) {
-  if (opName.rfind("aten", 0) == 0) {
     // builtin operators.
     Symbol symbol = Symbol::fromQualString(opName);
-    for (const auto& op : torch::jit::getAllOperatorsFor(symbol)) {
-      try {
-        // FIXME: This is temporary solution. We should at least refactor
-        // ``createStackForSchema`` to avoid throwing an error.
-        Stack stack = torch::jit::createStackForSchema(
-            op->schema(), args, kwargs, c10::nullopt);
-
-        auto& autogradContainer = DistAutogradContainer::getInstance();
-        if (autogradContainer.hasValidContext()) {
-          // Get all the ivalues for the operator.
-          auto ivalues =
-              torch::jit::last(stack, op->schema().arguments().size());
-
-          // Attach the appropriate autograd edges.
-          auto grad_fn = addSendRpcBackward(ivalues);
+    if (true) {
+      // if (symbol.is_aten()) {
+      Stack stack;
+      for (const auto& op : torch::jit::getAllOperatorsFor(symbol)) {
+        try {
+          // FIXME: This is temporary solution. We should at least refactor
+          // ``createStackForSchema`` to avoid throwing an error.
+          stack = torch::jit::createStackForSchema(
+              op->schema(), args, kwargs, c10::nullopt);
 
-          // Record the send function in our current context.
-          auto& currentContext = autogradContainer.currentContext();
-          currentContext.addSendFunction(grad_fn);
+        } catch (std::runtime_error& e) {
+          LOG(INFO) << "Couldn't match schema: " << op->schema()
```

Comment:
This will print a lot of logs, I remember seeing ~20 overloaded versions of `aten::add`

Reply:
How many logs do you see when run tests locally?

Reply:
I didn't push the latest version here, I've changed this to `VLOG(1)` so it will print only when verbosity is on.